pwndbg/commands/patch.py

```diff
@@ -23,11 +29,11 @@
 
 @pwndbg.commands.ArgparsedCommand(parser)
 @pwndbg.commands.OnlyWhenRunning
-def patch(address, ins, quiet) -> None:
+def patch(address: gdb.Value | int, ins: str, quiet: bool) -> None:
     # Make sure that any gdb.Value object is converted to int
     address = int(address)
 
-    new_mem = asm(ins)
+    new_mem = bytearray(asm(ins))
```

Comment:
If it works, it works...

```suggestion
    new_mem = asm(ins)
```

Reply:
fixed!